### PR TITLE
9.0 product variant sale price

### PR DESCRIPTION
**Pull request by hurrinico**
Porting of @sergio-teruel module product variant sale price to version 9

### CODE REVIEW

File: product_variant_sale_price/models/product_product.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    _inherit = "product.product"
+
+    @api.multi
+    @api.depends('fix_price')
+    def _compute_lst_price(self):
+        for product in self:
+            price = product.fix_price or product.list_price
+            if 'uom' in self.env.context:
+                uom = product.uos_id or product.uom_id
+                price = uom._compute_price(price, self.env.context['uom'])
```
Comment (tafaRU) [inline product_variant_sale_price/models/product_product.py]: I think here you missed `product.uom_id.id` as first argument. Please also compare to https://github.com/sergio-teruel/product-variant/blob/8.0-PR-product_variant_sale_price/product_variant_sale_price/models/product_product.py#L32

File: product_variant_sale_price/models/product_product.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            product.lst_price = price
+
+    @api.multi
+    def _inverse_product_lst_price(self):
+        for product in self:
+            vals={}
+            if 'uom' in self.env.context:
+                uom = product.uos_id or product.uom_id
+                vals['fix_price'] = uom._compute_price(
+                    product.lst_price, self.env.context['uom'])
```
Comment (tafaRU) [inline product_variant_sale_price/models/product_product.py]: Same as above.

File: product_variant_sale_price/models/product_product.py (hunk 1 above)
Comment (tafaRU) [inline product_variant_sale_price/models/product_product.py]: Ok, in the PR there is, so never mind ;-)

File: product_variant_sale_price/models/product_product.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    @api.onchange('impact_price')
+    def _onchange_impact_price(self):
+        self.lst_price = self.list_price + self.impact_price
+
+    lst_price = fields.Float(
+        compute='_compute_lst_price',
+        inverse='_inverse_product_lst_price',
+    )
+    fix_price = fields.Float(string='Fix Price')
+    impact_price = fields.Float(
```
Comment (tafaRU) [inline product_variant_sale_price/models/product_product.py:62]: What' the reason for this?

File: product_variant_sale_price/tests/test_product_product.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+from openerp.tests.common import TransactionCase
+
+
+class TestProductProduct(TransactionCase):
+
+    def setUp(self):
+        super(TestProductProduct, self).setUp()
+        self.template_model = self.env['product.template']
+        self.product_model = self.env['product.product']
```
Comment (tafaRU) [inline product_variant_sale_price/tests/test_product_product.py:13]: test needs to be completed!

File: product_variant_sale_price/views/product_view.xml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                <field name="is_product_variant" invisible="1"/>
+                <field name="product_variant_count" invisible="1"/>
+            </field>
+            <xpath expr="//field[@name='list_price']" position="attributes">
+                <attribute name="attrs">{'readonly': [
+                                            ('is_product_variant', '=', False),
+                                            ('product_variant_count', '=', 1)
+                                        ]}</attribute>
+            </xpath>
+            <!--<xpath expr="//field[@name='list_price']" position="attributes">-->
```
Comment (tafaRU) [inline product_variant_sale_price/views/product_view.xml:20]: Please remove this commented part.

File: product_variant_sale_price/README.rst [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+   :target: https://runbot.odoo-community.org/runbot/205/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/server-tools/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```
Comment (tafaRU) [inline product_variant_sale_price/README.rst:29]: This part has been removed, please update it according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#bug-tracker

**Comment by pedrobaeza** [comment]
Are these comments going to be attended?

**Comment by tafaRU** [comment]
@pedrobaeza you can close it. Thanks.

**Pull request closed without merging**